src/naemon/configuration.c

```diff
@@ -1455,7 +1497,7 @@ int pre_flight_object_check(int *w, int *e)
 		}
 
 		/* check to see if there is at least one contact/group */
-		if (temp_service->contacts == NULL && temp_service->contact_groups == NULL) {
+		if (temp_service->contacts == NULL && temp_service->contact_groups == NULL && global_service_event_handler == NULL) {
```

Comment:
That should be probably `global_service_notification_handler`

Reply:
thanks, you are totally right